src/plan_bot.py

```diff
@@ -97,20 +95,18 @@ def process_post(
                 did_reply = reply(post, reply_string, send=send, simulate=simulate)
 
                 if did_reply and not skip_tracking:
-                    
+
                     posts_db.document(post.id).set(
                         {
-                            # TODO add more info about the match here
                             "replied": True,
                             "type": post.type,
-                            "post_author": "/u/" + post.author,
+                            "post_author": "/u/" + post.author.name,
```

Comment:
Reddit properties are often other instances, rather than strings https://praw.readthedocs.io/en/latest/code_overview/models/submission.html so had a couple things to fix down here

Reply:
Yeah, I think I'm going to do the same with the subreddit display name.  I was taking attribute names from the pushshift API without thinking that they likely do some processing on their end before returning the data.  I may add a field for the subreddit_display_name to our metadata so we can have the ID and the human-readable name when scanning the logs.